### PR TITLE
Add enable vss log and get event logs from Windows guest 

### LINKED ISSUE #60: [v1.2][Bug] [Windows] Get log files when quiesce snapshot test failed
##### SUMMARY

Test case "check_quiesce_snapshot" failed sometime, we need to get log files from guest OS for debugging when quiescing guest OS failed.

##### ISSUE TYPE
- Bug Report

##### ANSIBLE VERSION

```paste below



```

##### CONFIGURATION

```paste below



```

##### ENVIRONMENT / OS

Windows 10 and ESXi 7.0GA.

##### ACTUAL RESULTS

```
2021-06-10 10:28:12,010 | Failed at Play [check_quiesce_snapshot] ********************

2021-06-10 10:28:12,010 | TASK [Create snapshot 'quiesce_snapshot_1623320739' on '70ga_win_10_64_nvme_efi_ansible'] 
task path: /root/ansible-vsphere-gos-validation/common/vm_take_snapshot.yml:4
fatal: [localhost]: FAILED! => {
    "changed": false,
    "invocation": {
        "module_args": {
            "datacenter": "70GA",
            "description": "",
            "folder": "/70GA/vm",
            "hostname": "xx.xx.xx.xx",
            "memory_dump": false,
            "moid": null,
            "name": "70ga_win_10_64_nvme_efi_ansible",
            "name_match": "first",
            "new_description": null,
            "new_snapshot_name": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "port": 443,
            "proxy_host": null,
            "proxy_port": null,
            "quiesce": true,
            "remove_children": false,
            "snapshot_name": "quiesce_snapshot_1623320739",
            "state": "present",
            "use_instance_uuid": false,
            "username": "Administrator@vsphere.local",
            "uuid": null,
            "validate_certs": false
        }
    },
    "msg": "An error occurred while quiescing the virtual machine. See the virtual machine's event log for details."
}
```

### PR DESCRIPTION
**Pull request by Tomorrow9**
Fixes #60

### CODE REVIEW

File: windows/utils/win_enable_vss_log.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  loop:
+    - {key: '^#log = true', value: 'log = true'}
+    - {key: '^#vmvss.level = debug', value: 'vmvss.level = debug'}
+    - {key: '^#vmvss.handler = vmx', value: 'vmvss.handler = vmx'}
+  delegate_to: "{{ vm_guest_ip }}"
+
+# Restart VMware tools service
+# - include_tasks: win_execute_cmd.yml
+#   vars:
+#     win_powershell_cmd: "Restart-Service -Force -Name VMTools"
```
Comment (keirazhang) [inline windows/utils/win_enable_vss_log.yml:35]: If this task is not required, I think we can remove it. The other looks good to me. Thanks!

Comment (Tomorrow9) [reply windows/utils/win_enable_vss_log.yml:35]: Sure, when testing done, I'll leave or remove the commented lines. Thanks.

**Comment by Tomorrow9** [comment]
Test passed on Windows Server vNext build.
```
TASK [Check if get expected config in vmx] *************************************
task path: /root/ansible-vsphere-gos-validation/windows/check_quiesce_snapshot/check_vmx_disk_enable_uuid.yml:12
ok: [localhost] => {
    "changed": false,
    "msg": "'disk.EnableUUID = TRUE' is in VM vmx file."
}
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'

TASK [Set fact of the remote path in guest OS] *********************************
task path: /root/ansible-vsphere-gos-validation/windows/check_quiesce_snapshot/check_quiesce_snapshot.yml:19
ok: [localhost] => {
    "ansible_facts": {
        "quiesce_scripts_freeze_thaw_path": "C:\\Windows\\",
        "quiesce_scripts_tools_path": "C:\\Program Files\\VMware\\VMware Tools\\backupScripts.d\\"
    },
    "changed": false
}
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'
...
TASK [Get vmware.log from ESXi host] *******************************************
task path: /root/ansible-vsphere-gos-validation/windows/check_quiesce_snapshot/collect_vss_logs.yml:5
changed: [localhost -> cpdbj-214.eng.vmware.com] => {
    "changed": true,
    "checksum": "93a1552599afed61b3533763bc61035b4535abfc",
    "dest": "/root/ansible-vsphere-gos-validation/logs/70U2_winsrv_pvscsi_efi_ansible/2021-08-02-06-16-16/check_quiesce_snapshot/vmware.log",
    "md5sum": "eeec4957b22356a9759aaf38d19453fa",
    "remote_checksum": "93a1552599afed61b3533763bc61035b4535abfc",
    "remote_md5sum": null
}
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'

TASK [Display the result of getting vmware.log file] ***************************
task path: /root/ansible-vsphere-gos-validation/windows/check_quiesce_snapshot/collect_vss_logs.yml:14
ok: [localhost] => {
    "get_vmware_log": {
        "changed": true,
        "checksum": "93a1552599afed61b3533763bc61035b4535abfc",
        "dest": "/root/ansible-vsphere-gos-validation/logs/70U2_winsrv_pvscsi_efi_ansible/2021-08-02-06-16-16/check_quiesce_snapshot/vmware.log",
        "failed": false,
        "md5sum": "eeec4957b22356a9759aaf38d19453fa",
        "remote_checksum": "93a1552599afed61b3533763bc61035b4535abfc",
        "remote_md5sum": null
    }
}
Read vars_file '{{ testing_vars_file | default('../../vars/test.yml') }}'

TASK [Set fact of the saved event log files] ***********************************
task path: /root/ansible-vsphere-gos-validation/windows/check_quiesce_snapshot/collect_vss_logs.yml:18
ok: [localhost] => {
    "ansible_facts": {
        "application_event_log": "C:\\application.evtx",
        "system_event_log": "C:\\system.evtx"
    },
    "changed": false
}
```

**Review by keirazhang — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**